testutils/resourcebuilder/machine/v1beta1/aws_provider_spec_test.go

```diff
@@ -0,0 +1,100 @@
+package v1beta1
+
+import (
+	"testing"
+
+	machinev1beta1 "github.com/openshift/api/machine/v1beta1"
+	"k8s.io/utils/ptr"
+)
+
+// ------------- DNM: FOR DEMONSTRATION ONLY ----------------
```

Comment:
Why DNM? Having tests isn't harmful is it?

Reply:
I can remove the comment if we are happy with them. Or do we want them to use ginkgo?

Reply:
Would prefer ginkgo if you have the time to convert them across?

Reply:
Done